packages/blockchain-extension/templates/javaTestSmartContractTemplate.ejs

```diff
@@ -25,11 +25,18 @@ import java.nio.file.Paths;
 import java.util.Arrays;
 <%_ } _%>
 import java.util.concurrent.TimeoutException;
+<%_ if (isPrivateData) { _%>
+import java.util.HashMap;
+import java.util.Map;
```

Comment:
It looks like only `HashMap` is being used - do we still need to import `Map`?

Reply:
Map is an interface and HashMap is one of it's possible implementing classes.
The sdk has Map as input for that method: https://github.com/hyperledger/fabric-gateway-java/blob/2587c711772917188fa4198736dcad0ae1d7be92/src/main/java/org/hyperledger/fabric/gateway/Transaction.java#L53

It would work, but it's misleading because the user would not necessarily know that they could use other classes there.